src/main/scala/devices/tilelink/Zero.scala

```diff
@@ -20,6 +20,7 @@ class TLZero(address: AddressSet, beatBytes: Int = 4)(implicit p: Parameters)
       executable = true,
       mayDenyGet = false,
       mayDenyPut = false),
+    minLatency = 0,
     beatBytes = beatBytes,
```

Comment:
@terpstra based on your comment above I changed this to indicate to the parent class that it actually has `minLatency` of 0.

Reply:
But it doesn't have a minLatency of 0, it has a minLatency of 1, because of the Queue in the implementation below.

I think previous to this PR all the existing implementations had to conform to minLatency=1 because that was the behavior specified in the base class.

Reply:
Ok, I was confused that @terpstra said it had minLatency=0...